### DISCUSSION TITLE
Burning, Smart Contract, PoH hybrid PoW

### DISCUSSION THREAD

**Discussion by CHIPofficial** [dogecoin/dogecoin#2427] [General]
I want to ask @rnicoll if it is possible to do a burning element to Dogecoin so the price of Dogecoin can go up easier like EIP-1559. I also read about Solana which has a PoH hybrid PoS, but we have PoW so we can do a PoH hybrid PoW. I what to use PoH because it is fast very fast. Solana is 65,000 TPS. Solana whitepaper: https://solana.com/solana-whitepaper.pdf. Also, I want Dogecoin to be a payment currency and a useable currency, so it is easier to build Dapp without using a Dogethereum bridge.

**Comment by tjstebbing** [comment]
You're welcome to buy dogecoin and burn your wallet if you like :) dogecoin is dogecoin.

**Comment by ScalpedMiner** [comment]
Correct me if I'm wrong but I don't think making a cryptocurrency more scarce would make it a better currency. It may make it a better store of value but a currency has to be widely available and readily accessible. Many argue that EIP-1559 is an upgrade just so Ethereum can de-thrown Bitcoin as No.1 Crypto in the market. I think we need to let Dogecoin be Dogecoin and not copy other networks, Devs have a plan, and we should trust that plan. No need to try and change all of the work being done since 2019, just to create more work to turn dogecoin into something other than dogecoin.

**Comment by ReverseControl** [comment]
@CHIPofficial Long answer, no.
